Create ETL to move drug list from CAMP to TAC

spamhurts opened the pull request:
#### Rationale
Preliminary work before adding treatment orders and distribution workflow to TAC

Jira issue # TT-11

spamhurts committed:
- Updated snprc_ehr module version to 24.001
- Updated snprc_ehr schema to include tables for the therapy drug list (formulary) tables and related lookups
- Added source queries to transfer drug list and lookup table data from CAMP
- Created multi-step ETL to transfer formulary and related lookup table data

RameshRapa commented on snprc_ehr/resources/source_queries/therapy/create_v_delete_therapy_formulary_and_lookups.sql:
```diff
@@ -0,0 +1,59 @@
+CREATE VIEW labkey_etl.v_delete_therapy_formulary_and_lookups AS
```
Hi Terry. Can i please know the purpose of this delete therapy etl? why do we use this and the functionality around this ?

spamhurts committed:
- Updated metadata for the therapy drug list (formulary) tables and related lookups to allow nulls in the RowId and ObjectID columns during data entry
- Added trigger scripts to maintain RowIds and ObjectIds in the formulary and related tables
- Added Java helper method to retrieve the next RowId value from the passed in table name
